ivy/functional/ivy/losses.py

```diff
@@ -381,4 +382,47 @@ def sparse_cross_entropy(
     true = ivy.one_hot(true, pred.shape[axis])
     return ivy.cross_entropy(
         true, pred, axis=axis, epsilon=epsilon, reduction=reduction, out=out
-    )
+
+
+        )
+def mse_loss(true : Union[ivy.Array, ivy.NativeArray],pred : Union[ivy.Array, ivy.NativeArray],axis : int=-1,out : Optional[ivy.Array]=None)-> ivy.Array:
+    """
+    Compute the mean square error
+
+    Parameters
+    ----------
+
+    true
+        input array containing true labels
+    pred
+        input array containing predicted labels
+    axis
+        the axis along which mse is computed. By default, it is the last axis (-1)
+    out
+        to write the result
+
+    Returns
+    -------
+    ret
+        the mean square loss between two arrays
+
+    Example
+    -------------------
+    >>x = [1,1,1]
+    >>y = [0,0,0]
+    >>z = ivy.mse_loss(x,y)
+    >>print(z)
+    ivy.array([1])
+    """
+     
+    true = ivy.negative(true)
+    diff = ivy.add(pred,true,axis=axis)
+    s_diff = ivy.multiply(diff,diff,axis =axis)
+    s_diff = ivy.sum(s_diff,axis = axis)
+    mse_diff = s_diff/true.shape[-1]
+    return mse_diff
+
+
+
```

Comment:
Remove excess whitespace. There should be one line of whitespace at the end of the file.